File: openapi/code_samples/javascript/api_company_{cui}.js
const cui = "YOUR_CUI_HERE";
const currency = "YOUR_CURRENCY_HERE"; // Default: RON

fetch(`https://app.metricbiz.com/v1/api/companies/${cui}?currency=${currency}`)
    .then(response => response.json())
    .then(data => console.log(data))
    .catch(error => console.error('Error:', error));

File: openapi/code_samples/csharp/api_company_{cui}.cs
using System;
using System.Net.Http;
using System.Threading.Tasks;

public class Program
{
    private static readonly HttpClient httpClient = new HttpClient();

    public static async Task Main(string[] args)
    {
        var cui = "YOUR_CUI_HERE";
        var currency = "YOUR_CURRENCY_HERE"; // Default: RON

        var response = await httpClient.GetAsync($"https://app.metricbiz.com/v1/api//companies/{cui}?currency={currency}");

        if (response.IsSuccessStatusCode)
        {
            var content = await response.Content.ReadAsStringAsync();
            Console.WriteLine(content);
        }
        else
        {
            Console.WriteLine($"Error: {response.StatusCode}");
        }
    }
}
